node_modules/.package-lock.json

```diff
@@ -386,6 +386,84 @@
         "node": ">=6.9.0"
```

Comment:
- [ ] Kindly remove the node_modules file from your repository by adding it to your `.gitignore` file